termtosvg/data/templates/carbon_js.svg

```diff
@@ -0,0 +1,232 @@
+        <g id="icon-play">
+            <path d="M0 4l14 11-14 11z"/>
+        </g>
+        <g id="icon-pause">
+            <rect x="0" y="5" width="5" height="20"/>
+            <rect x="10" y="5" width="5" height="20"/>
+        </g>
+    </defs>
+    <rect id="terminalui" width="100%" height="100%" ry="4.5826941"/>
+    <circle cx="24" cy="23" r="7" class="fake-close-button"/>
+    <circle cx="44" cy="23" r="7" class="fake-minimize-button"/>
+    <circle cx="64" cy="23" r="7" class="fake-expand-button"/>
+    <svg id="screen" width="656" x="23" y="50" viewBox="0 0 656 204"
+         preserveAspectRatio="xMidYMin meet">
+        <!-- The content of this SVG tag does not serve any purpose and will be replaced
+        by the code generated by termtosvg at animation rendering time -->
+        <defs>
+            <text id="awesome" fill="#f8f8f2" lengthAdjust="spacingAndGlyphs" textLength="168">
+                termtosvg is awesome!
+            </text>
+            <style type="text/css"><![CDATA[* {font-family: "DejaVu Sans Mono", monospace; font-style: normal; font-size: 14px}
+                text {dominant-baseline: text-before-edge}
+                .background {fill: #272822}]]></style>
+        </defs>
+        <rect class="background" height="100%" width="100%"/>
+        <use xlink:href="#awesome" x="0" y="0"/>
+        <use xlink:href="#awesome" x="8" y="17"/>
+        <use xlink:href="#awesome" x="16" y="34"/>
+        <use xlink:href="#awesome" x="24" y="51"/>
+        <use xlink:href="#awesome" x="32" y="68"/>
+        <use xlink:href="#awesome" x="40" y="85"/>
+        <use xlink:href="#awesome" x="48" y="102"/>
+        <use xlink:href="#awesome" x="56" y="119"/>
+        <use xlink:href="#awesome" x="64" y="136"/>
+        <use xlink:href="#awesome" x="72" y="153"/>
+        <use xlink:href="#awesome" x="80" y="170"/>
+        <use xlink:href="#awesome" x="88" y="187">
+            <animate attributeName="x" begin="0ms;anim.end" dur="10000ms"
+                     from="88" to="588" id="anim"/>
+        </use>
+    </svg>
+    <text id="timer">0:00/0:00</text>
+    <!-- Invisible, wider track to make frame seeking easier -->
+    <use xlink:href="#slider_wide_track" id="wide_track"/>
+    <!-- Visible track -->
+    <use xlink:href="#slider_track" id="track"/>
+    <use xlink:href="#slider_button" id="slider_1" x="0px">
+        <animate attributeName="x" begin="0ms;anim_slider_button.end" dur="0ms"
+                 from="0" to="0" id="anim_slider_button"/>
+    </use>
+    <use xlink:href="#icon-play" id="play-button"/>
+    <use xlink:href="#icon-pause" id="pause-button"/>
+    <script type="text/javascript"><![CDATA[
+var terminal   = document.getElementById('terminal');
+var screen = terminal.getElementById('screen');
+var slider_1  = terminal.getElementById('slider_1');
+var anim_slider_button = document.getElementById('anim_slider_button');
+var style = getComputedStyle(terminal);
```

Comment:
Maybe use `window.getComputedStyle` here for consistency?

Reply:
Also, you don't need to create the `style` variable here. Instead you can do this:


```